scripts/scr_apothecary_ground/scr_apothecary_ground.gml

```diff
@@ -8,8 +8,9 @@ function calculate_full_chapter_spread(turn_end=true){
 	var tech_spread = {};
 	var apoth_spread = {};
 	var unit_spread = {};
+	var maintenance_burden=0;
     for(var company=0;company<11;company++){
-    	for (var v=1;v<500;v++){
+    	for (var v=0;v<500;v++){
```

Comment:
Should this initialize from 0-500 inclusive instead of 0-499? Switching to `array_length` here avoids hard-coded numbers, reducing off-by-one issues related to these arrays.

```suggestion
    	for (var v=0; v < array_length(obj_ini.name[company]); v++) {
```